src/typing/azdata.proposed.d.ts

```diff
@@ -738,6 +740,10 @@ declare module 'azdata' {
 		flavor: string;
 	}
 
+	export interface QueryExecutionOptions {
+		options: Map<string, any>;
```

Comment:
Why do we need an entire structure for this? Can't the argument passed just be a map?